# Fix jinja2 to build wwith plain old distutils.  Setuptools is not supported and is a pain to get working
--- a/setup.py	2016-03-16 15:01:14.425354992 -0600
+++ b/setup.py	2016-03-16 15:01:40.073303572 -0600
@@ -35,7 +35,7 @@
 .. _Jinja2 webpage: http://jinja.pocoo.org/
 .. _documentation: http://jinja.pocoo.org/2/documentation/
 """
-from setuptools import setup
+from distutils.core import setup
 
 
 setup(
